src/odemis/acq/stream/_helper.py

```diff
@@ -1550,6 +1551,9 @@ def _updateAcquisitionTime(self) -> None:
         """
         # Override standard method (on LiveStream), which checks whether the acquisition time has
         # changed "a lot", and in such case stop and restart the acquisition.
+        # Here, we would need to restart both the scanner and detector, which is more complex...
+        # especially because this function is called just after active is set, which would cause an
+        # immediate restart.
```

Comment:
This comment raises more questions for me than that it explains anything. For instance why would the scanner and detector need to be restarted, and compared to what is it more complex?